### PR TITLE
501 bug eyewitness tooltip

### PR DESCRIPTION
**Pull request by michaeljpigott**
**Bug fix:** EyeWitness tooltip

**Pull request description:** 
 This bug fix adds a tooltip to the EyeWitness tool, giving users better information about what the tool is for, and how to use it. 

**Changes made:** 
* Added title and description_userguide variables to eyewitness.tsx. 
* Imported UserGuide component into eyewitness.tsx. 
* Updated title (to camel case) and description of EyeWitness on the Tools page. 

I performed a style check and used Prettier to update eyewitness.tsx.

### CODE REVIEW

File: src/components/eyewitness/eyewitness.tsx
Comment (bradleybogg) [inline src/components/eyewitness/eyewitness.tsx:1]: Good work.

**Review by bradleybogg — COMMENTED** [review]
Just need to confirm that the Eyewitness change to the RouteWrapper will only affect object name printing. No logical change.

File: src/components/RouteWrapper.tsx [hunk 1]
```diff
@@ -351,10 +351,10 @@ export const ROUTES: RouteProperties[] = [
         category: "Network Scanning and Enumeration",
     },
     {
-        name: "Eyewitness",
+        name: "EyeWitness",
         path: "/tools/eyewitness",
         element: <Eyewitness />,
-        description: "Eyewitness takes screenshot of important websites.",
+        description: "EyeWitness scans listed URLs and provides a HTML report about them, including screenshots",
```
Comment (bradleybogg) [inline src/components/RouteWrapper.tsx:357]: Super minor change. Please add a full stop at the end. Thank you!

- [ ] Add a full stop at the end.

**Review by bradleybogg — CHANGES_REQUESTED** [review]
Hi @michaeljpigott,

Very minor change.

Thank you!

Bradley

**Comment by bradleybogg** [comment]
Hi @michaeljpigott,

Please also see this resource that I learnt from and you might too.

https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

File: src/components/RouteWrapper.tsx (hunk 1 above)
Comment (michaeljpigott) [reply src/components/RouteWrapper.tsx]: Thank you, I added the full stop.

**Review by bradleybogg — APPROVED** [review]

**Review by bradleybogg — APPROVED** [review]

**Review by bradleybogg — APPROVED** [review]

**Pull request merged**